### PR TITLE
feat(form): form组件增加自动换行

### PR DESCRIPTION
**Pull request by TC-twwang**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/ui-vue/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/action-sheet/index.js [hunk 1]
```diff
@@ -10,7 +10,47 @@
 *
 */
 
-export const visibleHandle = (emit) => () => {
+export const setSheetStyle = ({ state, props }) => () => {
+  if (props.isContent) {
```
Comment (shenjunjian) [inline src/action-sheet/index.js]: 考虑为isContent 这个属性，改个名字！

通常组件的属性起名方式和 自己的编码时的变量名是不一样的。    组件属性需要是名词或描述性质的词，比如 position, content-position 这样

Comment (TC-twwang) [reply src/action-sheet/index.js]: 已改为content-position

File: src/form-item/index.js [hunk 2]
```diff
@@ -47,28 +47,21 @@ export const computedLabelStyle = ({ props, state }) => () => {
...
-
-  if (state.form.labelPosition === POSITION.Top || state.form.inline) {
-    return result
-  }
-
-  if (!label && !props.labelWidth && state.isNested) {
-    return result
+  
+  if (state.form.labelPosition === POSITION.Top) {
+    return result.width = `100%`
```
Comment (zzcr) [inline src/form-item/index.js]: 这里模板字符串中没有变量，直接使用普通字符串就行

Comment (TC-twwang) [reply src/form-item/index.js]: 已修改

**Review by zzcr — APPROVED** [review]

**Pull request merged**